airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -579,7 +579,7 @@ def transform_empty_string_to_none(instance: Any, schema: Any):
 
 class IncrementalRestSalesforceStream(RestSalesforceStream, ABC):
     state_checkpoint_interval = 500
-    STREAM_SLICE_STEP = 120
+    STREAM_SLICE_STEP = 30
```

Comment:
what is the impact of changing this? Do other salesforce syncs now take longer?

Reply:
By decreasing the `window` we make  responses smaller and thus iterate over them quicker.
According to screen it takes roughly the same time to sync (or even slightly less):
First sync was performed using `2.0.13-dev.b206a930`, next 2 using `2.0.9`
![image](https://github.com/airbytehq/airbyte/assets/36314070/0c89fdfb-65de-4dfc-8c39-2755065415ac)

Reply:
👍🏼